americangut/util.py

```diff
@@ -492,7 +492,7 @@ def clean_and_reformat_mapping(in_fp, out_fp, body_site_column_name,
             body_site = body_site.split('_', 1)[-1].replace("_", " ")
         elif body_site.startswith('UBERON:'):
             body_site = body_site.split(':', 1)[-1]
-        elif body_site in ['NA', 'unknown', '', 'no_data', 'None']:
+        elif body_site in ['NA', 'unknown', '', 'no_data', 'None', 'Unknown']:
```

Comment:
Since I've actually tested this specific fix, I'd rather merge this now. Let's revisit this later and we can change it to Jamie's suggestion if we think that is the way to go. Thanks!